vignettes/polygon_intersection.Rmd

```diff
@@ -143,7 +154,7 @@ blue$area <- as.numeric(sf::st_area(blue))
   left_join(blue_pink_norm_false, by = "idblue"))
 ```
 
```

Comment:
Would be helpful to have source and target polygons (blue and/or pink) identified for each use case.  I noticed this is added in line 304, but would be helpful in other spots.